src/App.js

```diff
@@ -1,12 +1,66 @@
-import React from 'react';
+import React, {useEffect, useState} from "react";
+import axios from 'axios';
 import './App.css';
+import PokemonCards from "./components/pokemon/PokemonCards";
+import Button from "./components/button/Button";
 
-function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+
+
+// Het resultaat opslaan in state.
+// Een console log maken van het resultaat
+// Bekijk wat je terug krijgt van de request in de dev tools
+
+function  App() {
+    const [pokemon, setPokemon] = useState  ([])
+    const [endpoint, setEndpoint] = useState (`https://pokeapi.co/api/v2/pokemon/jigglypuff`)
+    const [state, setState ]= useState ([])
+
+
+    useEffect ( () => {
+        async function getPokemonData() {
+            try {
+                const result = await axios.get  (`${endpoint}`)
```

Comment:
Deze kan iets simpeler, omdat de string die je opbouwt niet anders is dan `endpoint` zelf
```suggestion
                const result = await axios.get(endpoint);
```